core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/keymanager/imports/ImportKeystoresHandler.java

```diff
@@ -169,7 +165,8 @@ public void handle(RoutingContext context) {
           final BlsArtifactSigner signer =
               decryptKeystoreAndCreateSigner(jsonKeystoreData, password);
           // 2. write keystore file to disk
-          createKeyStoreYamlFileAt(pubkey, jsonKeystoreData, password);
+          new KeystoreConfigurationFilesCreator(keystorePath, pubkey, jsonKeystoreData)
```

Comment:
We have a KeystoreFileManager so it probably make sense for the create to part of that

Reply:
refactored to KeystoreFileManager.